src/components/User.vue

```diff
@@ -45,7 +46,9 @@ watchEffect(() => {
         </div>
         <div class="m-4">
           <a
-            :href="_explorer(space ? space.network : '1', address)"
+            :href="
+              _explorer(proposal?.network || space?.network || '1', address)
```

Comment:
Here also

Reply:
@bonustrack we need space here cause, This User modal is used in many places, for example on delegate page we need to redirect them to based on space network and on space page we are redirecting them to mainnet etc